csi.proto

```diff
@@ -644,7 +644,7 @@ message ControllerPublishVolumeResponse {
   // The contents of this field SHOULD NOT contain sensitive
   // information.
   // The contents of this field SHOULD NOT be used for uniquely
-  // identifying a volume. The `id` alone should be sufficient to
+  // identifying a volume. The `id` alone SHOULD be sufficient to
```

Comment:
`id` should be `volume_id` here, right?  I guess we missed that in the `id`->`volume_id` PR

Reply:
Yes, good catch will fix!